scalikejdbc-core/src/main/scala/scalikejdbc/ConnectionPoolSettings.scala

```diff
@@ -10,5 +10,6 @@ case class ConnectionPoolSettings(
   validationQuery: String = null,
   connectionPoolFactoryName: String = null,
   driverName: String = null,
-  warmUpTime: Long = 100L)
+  warmUpTime: Long = 100L,
+  timeZone: String = null)
```

Comment:
I don't understand your point. Do you mean nullable fields are easier to construct because it requires fewer key strokes?

I searched `timeZone` field usage, and found 2 occurrences. One is to construct ConnectionPoolSettings from a config
```
timeZone = configMap.get("timeZone").orNull[String]
```

and Another is getting `timeZone` from a settings

```scala
    val timeZoneSettings = Option(settings.timeZone).fold(TimeZoneSettings()) { timeZone =>
      TimeZoneSettings(true, java.util.TimeZone.getTimeZone(timeZone))
    }
```

I felt that `Option[String]` is better by just looking these two. At the second occurrence, we might code wrongly if we forget it's nullable.